packages/@aws-cdk/aws-apigateway/package.json

```diff
@@ -244,6 +244,8 @@
       "docs-public-apis:@aws-cdk/aws-apigateway.JsonSchema.contains",
       "props-default-doc:@aws-cdk/aws-apigateway.JsonSchema.contains",
       "docs-public-apis:@aws-cdk/aws-apigateway.MethodProps",
+      "docs-public-apis:@aws-cdk/aws-apigateway.JsonSchema.default",
+      "props-default-doc:@aws-cdk/aws-apigateway.JsonSchema.default",
```

Comment:
These shouldn't be required.

Reply:
thx @nija-at I thought the same but the build was failing without those.

Reply:
ah just the default on you mean

Reply:
Neither should be required. One is stating that the property has no documentation and the other is stating that the default is not documented. You have both specified.

Reply:
Than the build should fail. I show you.

Reply:
Hi @nija-at . Indeed it passed. I am sure it failed before without that two lines. Anyway. Thank you for baring with me :). Cheers